ArmPkg/Include/IndustryStandard/ArmFfaSvc.h

```diff
@@ -10,7 +10,19 @@
 #define __ARM_FFA_SVC_H__
 
 #define ARM_SVC_ID_FFA_VERSION_AARCH32             	0x84000063
+#define ARM_SVC_ID_FFA_MSG_SEND_DIRECT_REQ_AARCH32      0x8400006F
+#define ARM_SVC_ID_FFA_MSG_SEND_DIRECT_RESP_AARCH32     0x84000070

```

Comment:
These IDs are defined in Arm [FF-A v1.0](https://developer.arm.com/documentation/den0077/latest)